frontend/src/components/Wallet.js

```diff
@@ -0,0 +1,184 @@
+// SPDX-License-Identifier: MIT
+// SDPX-FileCopyrightText: 2021 Que Le <b.le@tu-berlin.de>
+
+
+import appwriteApi from "../api/appwriteApi";
+import { useHistory } from "react-router-dom";
+import { useState } from "react";
+import detectEthereumProvider from "@metamask/detect-provider";
+const { utils } = require( "ethers" );
```

Comment:
For myself: What benefit does `require` have over import? Is there one?

Reply:
https://stackoverflow.com/a/46677972/7838440 answers your question pretty well. We should not use require, but import instead. You can run `npm i ethers` to add it to the package.json and your downloaded node_modules